lib/service/mixPanel_client_service.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+import 'dart:developer';
+
+import 'package:autonomy_flutter/common/environment.dart';
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/service/account_service.dart';
+import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/util/string_ext.dart';
+import 'package:crypto/crypto.dart';
+import 'package:mixpanel_flutter/mixpanel_flutter.dart';
+
+class MixPanelClientService {
+  final AccountService _accountService;
+  MixPanelClientService(this._accountService);
+
+  late Mixpanel mixpanel;
+  Future<void> initService() async {
+    final defaultDID = (await (await _accountService.getCurrentDefaultAccount())
+        ?.getAccountDID()) ??
+        'unknown';
+    final hashedUserID = sha256.convert(utf8.encode(defaultDID)).toString();
+
+    final defaultAddress = await (await _accountService.getCurrentDefaultAccount())
```

Comment:
looks like duplicated code here, can we create a local variable for `await _accountService.getCurrentDefaultAccount()`?